src/include/cell.i

```diff
@@ -897,11 +990,14 @@ __wt_cell_unpack_dsk(
          * If there isn't any value validity window (which is what it will take to get to a
          * zero-length item), the value must be stable.
          */
+        unpack->durable_start_ts = WT_TS_NONE;
```

Comment:
There is some logic further down this function for handling cells read from a previous run (so their transaction ids can't be used). We clear timestamps in some of these cases so you probably need to check whether you need to add something here too.

Reply:
I've added a FIXME comment.

Reply:
I don't see a `FIXME` for this and I don't think it is addressed.